#!/usr/bin/env python
PACKAGE = "aero_control"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("X_Position", double_t, 0, "X_Position", 0, -5, 5)
gen.add("Y_Position", double_t, 0, "Y_Position", 0, -5, 5)



exit(gen.generate(PACKAGE, "BaseServoPoint", "BaseServoPoint"))
